tests/integration/targets/aci_interface_policy_leaf_policy_group/tasks/main.yml

```diff
@@ -336,7 +336,9 @@
     link_level_policy: linklevelpolicy
     fibre_channel_interface_policy: fiberchannelpolicy
     state: present
+    port_channel_policy: ''
```

Comment:
You are modifying an existing test case. Which might impact other test cases. You should add a specific test for this failure check.